src/frontend/src/lib/components/ui/Button.svelte

```diff
@@ -0,0 +1,9 @@
+<script lang="ts">
+	export let color: 'primary' | 'secondary' = 'primary';
+	export let type: 'submit' | 'reset' | 'button' = 'submit';
+	export let disabled = false;
+</script>
+
+<button class={`${color} block flex-1`} {type} {disabled} on:click>
```

Comment:
If you have a better idea than `color`, go for it. Not good at naming today.

Reply:
ahahah honestly not really, i search around and it is mostly called either `type` (but we already use it) or `hierarchy` (and I don't like it)

for now, i change it to `style`, if we really don't like it, we change it after